scriptworker/test/test_worker.py

```diff
@@ -254,3 +260,296 @@ def _mocker_run_tasks_helper(mocker, exc, func_to_raise):
     mocker.patch.object(worker, "complete_task", new=noop_async)
     status = await worker.run_tasks(context)
     assert status == context.config['task_max_timeout_status']
+
+
+_MOCK_CLAIM_WORK_RETURN = {
```

Comment:
Same with these. If these are potentially useful to tests for other modules, these might belong in `test/__init__.py`

Reply:
I'm nervous about putting some of these in `__init__.py`, since they're not complete.
For example, if I moved `_MOCK_CLAIM_WORK_RETURN` to `__init__.py`, then either:
* Someone uses it in another test before realizing that the internal `tasks` value isn't fully filled out
* I add complete, correct mock data for `_MOCK_CLAIM_WORK_RETURN`, but not all of it is used. Upon future updates, we need to update (potentially unneeded) mock data, and it increases the "input" for the test (as a reader, it's less obvious what is needed to make the test pass)

My personal preference is to lean towards duplication if it makes individual tests smaller and easier to comprehend. When there are multiple tests that require full mocks, perhaps those full mocks could be shared?

Reply:
I didn't necessarily mean all of them, just the ones that make sense. The `create_{a,}sync` methods seem generic and useful enough that we could either share them or share them and replace `noop_{a,}sync` usage.